korau/src/commonMain/kotlin/com/soywiz/korau/format/mp3/minimp3/MiniMp3Program.kt

```diff
@@ -90,7 +90,7 @@ internal open class MiniMp3Program(HEAP_SIZE: Int = 0) : Runtime(HEAP_SIZE) {
     // typealias L12_subband_alloc_t = L12_subband_alloc_t
     // typealias L3_gr_info_t = L3_gr_info_t
     // typealias mp3dec_scratch_t = mp3dec_scratch_t
-    fun bs_init(bs: CPointer<bs_t>, data: CPointer<UByte>, bytes: Int): Unit {
+    fun bs_init(bs: CPointer<bs_t>, data: CPointer<UByte>, bytes: Int) {
```

Comment:
Note, since this code is autogenerated and I might need to update it in the future when further optimizations are done. We might want to update also here:
https://github.com/korlibs/ktcc/blob/f3d1cace49f6a74e3aad71a2cdcc22b877d3c783/src/commonMain/kotlin/com/soywiz/ktcc/gen/Kotlin.kt#L380